k8s/regions/frankfurt/test.yaml

```diff
@@ -0,0 +1,147 @@
+kubernetes:
+    app_name: "sumo-dev"
+    namespace: "sumo-test"
+    target_environment: "test"
+    nodeport_name: "sumo-nodeport"
+    secrets_name: "sumo-secrets-dev"
```

Comment:
This is going to specific to test, feels like this will need to be created outside of this area. Let's set this to sumo-secrets-test.